content/docs/components/serving/kfserving.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "KFServing"
+description = "Using KFServing for serving Models"
+weight = 2
++++
+
+## KFServing with Kubeflow
```

Comment:
Remove this heading. It's basically a duplicate of the page title, and it's confusing to the reader to see another heading immediately below the page title and description.